BMM.Core/Implementations/Analytics/AppCenterAnalytics.cs

```diff
@@ -30,27 +31,21 @@ public void LogEvent(string eventName, IDictionary<string, object> parameters)
             var user = _userStorage.GetUser();
             if (user != null)
             {
-                if (_remoteConfig.UseAnalyticsId)
```

Comment:
removing the if looks wrong

Reply:
I see you decided to remove the property UseAnalyticsId and assume that it's always true. Good thinking.
However, then we should never log the PersonId.